### ISSUE TITLE
Documentation for Windows can do with some clarification

### ISSUE THREAD

**Issue by StephanBijzitter** [jsontypedef/json-typedef-infer#12]
I downloaded the pre-built binary and while I did read that it was a pre-build binary, my brain didn't exactly connect the dots and I tried to use it as if it were an installer. Took me a good few minutes before the "Oooohhhhh!" moment finally came.

I'm not sure if this is possible, but when double clicking `jtd-infer.exe` a terminal/command-prompt window opens and it's blank. Perhaps a message could be added there?

Either way, thanks for this tool. Works like a charm!

**Comment by ucarion** [comment]
I can look into this, but it might be a challenge. The issue is that I'm
using a cross-compiler to generate the windows executable, and I'm not sure
it's easy to add windows-specific code.

Maybe a bit of documentation can cover this instead? Not sure where it'd
go. I'm also not sure what the conventional way to do things in
windows-land is. In unix systems, a direct cli executable is conventional.

**Comment by StephanBijzitter** [comment]
Yeah if I had downloaded the executable on my macbook I wouldn't have double-clicked it either haha, maybe a `README.txt` could be added to the `.zip` file for Windows, but I've never worked with Rust so I really wouldn't know
